kornia/color/rgb.py

```diff
@@ -169,6 +169,55 @@ def rgba_to_bgr(image: torch.Tensor) -> torch.Tensor:
     return rgb_to_bgr(x_rgb)
 
 
+def rgb_to_linear_rgb(image: torch.Tensor) -> torch.Tensor:
+    r"""Convert an sRGB image to linear RGB. Used in colorspace conversions.
```

Comment:
ok, given this function definition we should definitely name this function as `srgb_to_rgb`